rm_dbus:
  serial_port: "/dev/ttyUSB0"
  max_vel: 3
